### PR TITLE
Add imctools

### PR DESCRIPTION
**Pull request by drpatelh**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Addition of `imctools` to `bioconda`. Local tests with `bioconda-utils` passing on Linux. 

This is my first Python-based submission so it would be great if you can just double-check everything is ok. 

I was also wondering what the difference would be in using:
```
 script: pip install guzzle_sphinx_theme && $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

as opposed to :
```
script: pip install guzzle_sphinx_theme && python -m pip install --no-deps --ignore-installed .
```

Thanks in advance!

### CODE REVIEW

**Comment by drpatelh** [comment]
Sorry. Forgoto to ping @bioconda/core

**Comment by epruesse** [comment]
Please don't install `guzzle_sphinx_theme` into the `imctools` package. It should be a separate package (it is on pypi after all). If it is not available as conda package yet, please use the `conda skeleton pypi` tool to create a package and submit it at conda-forge. You may ping `@bgruening` there to expedite the merge of your PR so that this one here can proceed.

Yes - this is a little more circuitous than you likely hoped. But packages should not contain things other than themselves - otherwise we run into clashes with file names already taken.

**Comment by dpryan79** [comment]
None of the sphinx stuff is needed (likewise, pip and flake8 aren't needed) and I've removed them. That's all for generating documentation in the package, which isn't part of the installation usually.

**Comment by drpatelh** [comment]
Apologies @epruesse. This was more out of naivety than anything else. I saw this being done for another package so I thought it would be acceptable.

Thanks @dpryan79 . I see now that the requirements come from `setup.py` and not `requirements.txt`. I did try using `noarch: python` as recently recommended but I was getting some build errors suggesting that a couple of `.so` libraries couldnt be found. I then found a found a post where you suggested to remove that line. However, I strongly suspect that was just down to the state of the recipe at the time.

Thank you very much.

File: recipes/imctools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - scipy
+    - pandas
+
+test:
+  imports:
+    - imctools
+
+about:
+  home: https://github.com/BodenmillerGroup/imctools
+  license: UNKNOWN
```
Comment (bgruening) [inline recipes/imctools/meta.yaml]: This seems to be a MIT license.

**Comment by epruesse** [comment]
Ah - must be some caching issue - the `changed files` link showed me the old `pip install ...` line. Thanks @dpryan79

**Comment by drpatelh** [comment]
Am I ok to merge this?

**Comment by bgruening** [comment]
Yes you can! :)

**Pull request merged**